libs/qCC_io/AsciiFilter.cpp

```diff
@@ -70,14 +70,17 @@ AsciiOpenDlg* AsciiFilter::GetOpenDialog(QWidget* parentWidget/*=0*/)
 	return s_openDialog;
 }
 
-bool AsciiFilter::canLoadExtension(const QString& upperCaseExt) const
+
+AsciiFilter::AsciiFilter() :
+	FileIOFilter( {
+				  "_ASCII Filter",
```

Comment:
Is there a reason for the underscore before 'ASCII'?

Reply:
Oh, it seems it depends on whether the filter is defined in qCC_IO or plugins? Isn't it redundant with the 'FromPlugin' flag?

Reply:
> Isn't it redundant with the 'FromPlugin' flag? 

I was originally going to encode this info in the name - "_"  for core and "+" for core plugin - but making it explicit is better.

> Is there a reason for the underscore before 'ASCII'? 

The ID is supposed to be unique, so I was going to "reserve" "_" for internal (core) IDs. I will likely change the plugins to underscore once I start using the IDs.